content/tbb/tbb-9/contents.lr

```diff
@@ -6,4 +6,12 @@ seo_slug: using-tor-with-a-browser-besides-tor-browser
 ---
 description: 
 We strongly recommend against using Tor in any browser other than Tor Browser.
-Using Tor in another browser can leave you vulnerable without the privacy protections of Tor Browser.
+Tor Browser comes with default [security measures](https://tb-manual.torproject.org/security-settings/) that protect your digital footprint, and using Tor in another browser can leave you vulnerable without the privacy protection of Tor Browser.
+
+Some of the security measures you get to enjoy using Tor Browser include:
+1. Hiding your real IP address, so you can't be monitored. 
```

Comment:
I'd change "so you can't be monitored" to something like "Hiding your real IP address to preserve your online privacy".